server/events/tests/test_models.py

```diff
@@ -0,0 +1,36 @@
+from datetime import timedelta
+from django.utils import timezone
+from django.contrib.auth import models as auth_models
+from .shared_case import SharedTestCase
+from events.models import Event
```

Comment:
Importing classes from modules. 

If the name that you're importing is too long, feel free to use an alias like you did with `auth_models`.

Reply:
Fixed